File: examples/android/Notifications.kt
package com.example.aiapp

import android.content.Context
import androidx.core.app.NotificationCompat
import androidx.core.app.NotificationManagerCompat

// AI-driven notification example for Android
suspend fun sendAINotification(context: Context, message: String) {
    val content = AI.generateNotificationContent(message, "high")
    val notification = NotificationCompat.Builder(context, "channel_id")
        .setContentTitle(content.title)
        .setContentText(content.body)
        .setPriority(NotificationCompat.PRIORITY_HIGH)
        .setSmallIcon(R.drawable.ic_notification)
        .build()
    NotificationManagerCompat.from(context).notify(1, notification)
}

// Example usage
// sendAINotification(context, "Urgent: Meeting at 3 PM!")

File: examples/linux/notifications.py
import ai # type: ignore
import notify2 # type: ignore

# AI-driven notification example for Linux
async def send_ai_notification(message):
    content = await ai.generate_notification_content(message, urgency="critical")
    notify2.init("AIApp")
    notification = notify2.Notification(content.title, content.body, urgency=notify2.URGENCY_CRITICAL)
    notification.show()

# Example usage
# asyncio.run(send_ai_notification("Urgent: Meeting at 3 PM!"))